pCrunch/setup.py

```diff
@@ -22,35 +22,27 @@
 import sys
 from shutil import rmtree
 
-from setuptools import find_packages, setup, Command
+from setuptools import Command, setup, find_packages
 
 # Package meta-data.
-NAME = 'pCrunch'
-DESCRIPTION = 'Some tools for batch analysis of OpenFAST runs'
-URL = 'https://github.com/NREL/pCrunch'
-EMAIL = 'nikhar.abbas@nrel.gov'
-AUTHOR = 'Nikhar Abbas'
-REQUIRES_PYTHON = '>=3.4'
-VERSION = '0.1.1'
-
-# These packages are required for all of the code to be executed. 
+NAME = "pCrunch"
+DESCRIPTION = "Some tools for batch analysis of OpenFAST runs"
+URL = "https://github.com/NREL/pCrunch"
+EMAIL = "nikhar.abbas@nrel.gov"
+AUTHOR = "Nikhar Abbas"
```

Comment:
@JakeNunemaker, though flattering to have my name here, I'd say you should have more ownership over this than me. Same in the init function